posix-configs/SITL/init/ekf2/tailsitter

```diff
@@ -40,6 +40,8 @@ param set SYS_AUTOSTART 4010
 param set SYS_MC_EST_GROUP 2
```

Comment:
Change SYS_AUTOSTART to something more appropriate.

Reply:
Ah yeah, forgot about this